src/main/java/org/altadoon/gt6x/features/verticalmixers/VerticalMixers.java

```diff
@@ -0,0 +1,75 @@
+package org.altadoon.gt6x.features.verticalmixers;
+
+import gregapi.data.*;
+import gregapi.oredict.OreDictMaterial;
+import gregapi.tileentity.machines.MultiTileEntityBasicMachine;
+import gregapi.tileentity.machines.MultiTileEntityBasicMachineElectric;
+import gregapi.util.CR;
+import gregapi.util.UT;
+import net.minecraft.tileentity.TileEntity;
+import org.altadoon.gt6x.common.Config;
+import org.altadoon.gt6x.common.MTEx;
+import org.altadoon.gt6x.features.GT6XFeature;
+
+import static gregapi.data.CS.*;
+import static gregapi.data.CS.T;
+
+public class VerticalMixers extends GT6XFeature {
+    public static final String FEATURE_NAME ="Vertical Mixers";
+    @Override
+    public String name() {
+        return FEATURE_NAME;
+    }
+
+    @Override
+    public void configure(Config config) {}
+
+    @Override
+    public void preInit() {}
+
+    @Override
+    public void init() {
+        Class<? extends TileEntity> aClass;OreDictMaterial mat;
+
+        /* RU */
+        aClass = MultiTileEntityBasicMachine.class;
+        mat = MT.DATA.Kinetic_T[1]; MTEx.gt6xMTEReg.add("Vertical Mixer ("                         +mat.getLocal()+")", "Basic Machines"                      , 107, 20001, aClass, mat.mToolQuality, 16, MTEx.MachineBlock     , UT.NBT.make(NBT_MATERIAL, mat, NBT_HARDNESS,   7.0F, NBT_RESISTANCE,   7.0F, NBT_INPUT,   32, NBT_TEXTURE, "mixer_vertical", NBT_ENERGY_ACCEPTED, TD.Energy.RU, NBT_RECIPEMAP, RM.Mixer, NBT_INV_SIDE_IN, SBIT_U|SBIT_L, NBT_INV_SIDE_AUTO_IN, SIDE_LEFT, NBT_INV_SIDE_OUT, SBIT_D|SBIT_R, NBT_INV_SIDE_AUTO_OUT, SIDE_RIGHT, NBT_TANK_SIDE_IN, SBIT_U|SBIT_R, NBT_TANK_SIDE_AUTO_IN, SIDE_TOP, NBT_TANK_SIDE_OUT, SBIT_D|SBIT_L, NBT_TANK_SIDE_AUTO_OUT, SIDE_BOTTOM, NBT_ENERGY_ACCEPTED_SIDES, SBIT_B, NBT_PARALLEL, 4, NBT_PARALLEL_DURATION, T), "hPP","SRM","wPP",'M', OP.casingMachine.dat(mat), 'S', OP.stick.dat(mat), 'R', OP.rotor.dat(MT.StainlessSteel), 'P', OP.plate.dat(MT.StainlessSteel));
```

Comment:
Non auto tank in and out should be swapped.
So `NBT_TANK_SIDE_IN, SBIT_U|SBIT_L` and `NBT_TANK_SIDE_OUT, SBIT_D|SBIT_R`
it's weird that you input fluids in the same side you auto output items. This means if you use two fluid inputs and one item output you can't use auto out.
Same for the other machines

Reply:
fixed in [5e40da6](https://github.com/BorisMulder/gt6x/pull/13/commits/5e40da65a4f754164d84080243b923fa5b933b58)